infra/main.tf

```diff
@@ -0,0 +1,16 @@
+provider "digitalocean" {
+  # You need to set this in your .bashrc
+  # export DIGITALOCEAN_TOKEN="Your API TOKEN"
+  #
+}
+
+resource "digitalocean_droplet" "delta_droplet" {
+  ssh_keys           = [8187378]
+  image              = "ubuntu-16-10-x64"
```

Comment:
I don't see too much benefit there. I'd rather be on a newer version of the OS (which will hopefully ship with newer versions of Docker, etc). Since part of our deploy strategy will be to wholesale replace the server, there shouldn't be too much of a challenge in changing out the image to a newer one.